app/errors.py

```diff
@@ -0,0 +1,23 @@
+class VaccineError(Exception):
+    """Base class for vaccine-related errors."""
+
+    def __str__(self) -> str:
+        return "There was an error related to the vaccine."
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor is not vaccinated."""
+    def __str__(self) -> str:
+        return "The person is not vaccinated."
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when vaccine is expired."""
+    def __str__(self) -> str:
+        return "The person is not wearing a mask."
+
+
+class NotWearingMaskError(Exception):
+    """Raised when visitor is not wearing a mask."""
+    def __str__(self) -> str:
+        return "There was an error related to the vaccine."
```

Comment:
The error message in the `__str__` method of the `NotWearingMaskError` class doesn't seem to match the class name. The error message indicates that there was an error related to the vaccine, while the class name suggests that the error is about not wearing a mask. Please make sure that the error message accurately describes the error that the class represents. This point is from your checklist: 'Use descriptive error messages'.